packages/base/tsconfig.json

```diff
@@ -4,5 +4,5 @@
     "outDir": "lib",
     "rootDir": "src"
   },
-  "include": ["src/**/*", "src/schema/*.json", "src/_interface/*.json"]
+  "include": ["src/**/*", "src/schema/*.json", "src/_interface/*.json", "src/*.json"]
```

Comment:
I've temporarily added this as I've saved `keybindings.json` in `src/`, will be happy to move it to more valid location.